### DISCUSSION TITLE
Doing some tasks

### DISCUSSION THREAD

**Discussion by siakc** [Morganamilo/paru#1223] [General]
I like to do some tasks. Is there anyone who can onboard me for development?

**Comment by Morganamilo** [comment]
I've been struggling a bit with time this year so I've been way behind on keeping up with Paru issues. Next year should be a bit better so if you're still interested and are looking for something a bit more 1 to 1 rather than opening random PRs I'd really welcome that.

My contacts are:

- email: `morganamilo@archlinux.org`
- irc: `irc://irc.libera.chat:morganamilo`
- matrix: `@morganamilo:archlinux.org`

There's also the public `#paru` IRC channel on libera.

Message me on IRC or matrix, both are equally good for me. Though if you don't have a reliable always on IRC client I'd recommend matrix.

Tell me your experience and what sort of level of contribution you're looking to do and if you have any specific vision you want to push towards.
